CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [16.2.0](https://github.com/theforeman/puppet-puppet/tree/16.2.0) (2022-04-19)
+
+[Full Changelog](https://github.com/theforeman/puppet-puppet/compare/16.1.0...16.2.0)
+
+**Implemented enhancements:**
+
+- Add `server_jolokia_metrics_whitelist` parameter [\#826](https://github.com/theforeman/puppet-puppet/pull/826) ([alexjfisher](https://github.com/alexjfisher))
+- Change `server_puppetserver_profiler` and `server_puppetserver_metrics` defaults to true [\#825](https://github.com/theforeman/puppet-puppet/pull/825) ([alexjfisher](https://github.com/alexjfisher))
```

Comment:
I'm a bit torn on this one. The PRs are marked as `Enhancement`, but changing a default feels a bit like a breaking change to me?

What do you think @alexjfisher @ehelms?